src/modules/gnd_pos_control/gnd_pos_control_params.c

```diff
@@ -257,6 +257,32 @@ PARAM_DEFINE_FLOAT(GND_SPEED_TRIM, 3.0f);
  * @max 40
  * @decimal 1
  * @increment 0.5
- * @group FW TECS
+ * @group Rover Position Control
  */
 PARAM_DEFINE_FLOAT(GND_SPEED_MAX, 10.0f);
+
+/**
+ * Distance from front axle to rear axle
+ *
+ *
+ * @unit m
+ * @min 0.0
+ * @decimal 3
+ * @increment 0.01
+ * @group Rover Position Control
+ */
+PARAM_DEFINE_FLOAT(GND_WHEEL_BASE, 2.0f);
```

Comment:
We might as well change the parameter prefixes as well (GND -> RV). We can do it in a followup PR.